src/components/encrypt-video.js

```diff
@@ -9,9 +9,14 @@ module.exports = React.createClass({
   },
   getInitialState() {
     return {
-      seventyFivePercentDone: false
+      nextPercentDone: 0.25
     };
   },
+  videoGaEvent(props) {
+    props.category = props.category || "Video";
+    props.label = props.label || this.context.intl.formatMessage({id: this.props.video.title});
```

Comment:
Ah, filtering by url takes care of any issues, I was thinking you'd be looking across locales. Sounds good!